run-repeatedly.sh

```
@@ -41,14 +52,14 @@ cleanup_containers() {
     echo "Cleaning up containers after successful run $run_number..."
 
     # Stop and remove all containers
-    if [ "$(docker ps -q | wc -l)" -gt 0 ]; then
+    if [ "$($CONTAINER_RUNTIME ps -q | wc -l)" -gt 0 ]; then
         echo "Stopping running containers..."
-        docker stop $(docker ps -q) || true
+        $CONTAINER_RUNTIME stop $($CONTAINER_RUNTIME ps -q) || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)